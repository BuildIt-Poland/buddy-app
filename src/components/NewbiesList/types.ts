import { Newbie } from '../../../server/src/generated/schema-types';

export type NewbiesListProps = {
  newbies: Partial<Newbie>[];
};
