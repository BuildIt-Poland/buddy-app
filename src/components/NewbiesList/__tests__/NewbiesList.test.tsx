import React from 'react';
import { create } from 'react-test-renderer';

import NewbiesList from '../NewbiesList';

jest.mock('@material-ui/core/ListItem', () => 'ListItem');
jest.mock('@material-ui/core/Typography', () => 'AddIcon');
jest.mock('@material-ui/core/Box', () => 'Box');
jest.mock('../../Avatar', () => 'Avatar');

describe('Component - PlusButton', () => {
  test('renders correctly', () => {
    const component = create(
      <NewbiesList newbies={[{ name: ' foo', photo: 'url' }]} />
    );
    expect(component.toJSON()).toMatchSnapshot();
  });
});
