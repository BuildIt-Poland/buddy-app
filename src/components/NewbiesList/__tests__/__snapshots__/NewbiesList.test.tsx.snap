// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component - PlusButton renders correctly 1`] = `
Array [
  <AddIcon
    component="p"
    variant="h4"
  >
    <Box
      component="strong"
      fontWeight="fontWeightBold"
    >
      Newbies
    </Box>
  </AddIcon>,
  <ListItem
    button={true}
  >
    <AddIcon
      component="p"
      variant="h4"
    >
       foo
    </AddIcon>
    <Avatar
      imgSrc="url"
      type="small"
    />
  </ListItem>,
]
`;
